tests/test_dse.py

```diff
@@ -625,7 +625,7 @@ def test_tti_rewrite_aggressive_wmpi():
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('space_order,expected', [
-    (8, 152), (16, 268)
+    (8, 154), (16, 270)
```

Comment:
how?

Reply:
if you look back at the monkey patching PR, you may see that there we decreased them, while here we are simply restoring the pre-monkey-patching values. I this happens because in the DSE now we are back to where we were -- syntactically identical sub-expressions fail on == comparison because some of them have arguments which are sympy core objects, whereas with monkey patching everything was devito objects